lib/views/issueish-list-view.js

```diff
@@ -114,6 +115,7 @@ export default class IssueishListView extends React.Component {
   }
 
   renderMoreTile() {
```

Comment:
this is actually a link - we're taking you out of atom and into dotcom.  (The other instances I found are really playing the role of a button, though.)